### ISSUE TITLE
script/build fails with `cannot find module 'fs-extra'` error on OS X

### ISSUE THREAD

**Issue by adambuczynski** [atom/atom#12552]
Atom build 795e47f is acting up on OS X as well:

```
> script/build

module.js:457
    throw err;
    ^

Error: Cannot find module 'fs-extra'
at Function.Module._resolveFilename (module.js:455:15)
  at Function.Module._load (module.js:403:25)
  at Module.require (module.js:483:17)
  at require (internal/module.js:20:19)
  at Object.<anonymous> (/Users/Adam/Sites/forks/atom/atom/script/lib/clean-output-directory.js:1:74)
  at Module._compile (module.js:556:32)
  at Object.Module._extensions..js (module.js:565:10)
  at Module.load (/Users/Adam/Sites/forks/atom/atom/node_modules/coffee-script/lib/coffee-script/register.js:45:36)
  at tryModuleLoad (module.js:432:12)
  at Function.Module._load (module.js:424:3)
  at Module.require (module.js:483:17)
  at require (internal/module.js:20:19)
  at Object.<anonymous> (/Users/Adam/Sites/forks/atom/atom/script/build:20:30)
  at Module._compile (module.js:556:32)
  at Object.Module._extensions..js (module.js:565:10)
  at Module.load (module.js:473:32)
  at tryModuleLoad (module.js:432:12)
  at Function.Module._load (module.js:424:3)
  at Module.runMain (module.js:590:10)
  at run (bootstrap_node.js:394:7)
  at startup (bootstrap_node.js:149:9)
  at bootstrap_node.js:509:3
```

**Comment by as-cii** [comment]
This should be fixed in the latest commit on master and was caused by not running `script/bootstrap` as part of `script/build`.

Thanks for the report!

**Comment by adambuczynski** [comment]
I've tried running it with that commit though and it still didn't work.
Will recheck later this morning.

**Comment by adambuczynski** [comment]
It seems to be fixed as of ad24aae, thanks.
